src/graph/test/IndexTest.cpp

```diff
@@ -125,11 +125,13 @@ TEST_F(IndexTest, TagIndex) {
         std::string query = "SHOW TAG INDEX STATUS";
         auto code = client->execute(query, resp);
         ASSERT_EQ(cpp2::ErrorCode::SUCCEEDED, code);
-        std::vector<uniform_tuple_t<std::string, 2>> expected{
-            {"single_person_index", "SUCCEEDED"},
-            {"multi_person_index",  "SUCCEEDED"},
-        };
-        ASSERT_TRUE(verifyResult(resp, expected));
+        /*
```

Comment:
I think you can try 3 time checking.

Reply:
> I think you can try 3 time checking.

Because the final index data is not check at here, I don't think it is necessary to wait for the execution to complete.